### DISCUSSION TITLE
Need Wanbo x1 and x1 pro projector codes

### DISCUSSION THREAD

**Discussion by Absarking** [probonopd/irdb#174] [Code requests]
Hello everyone, I'm Absar I Need Wanbo x1 and x1 pro codes

**Comment by Absarking** [comment]
And It is a projector

**Comment by probonopd** [comment]
Hello. Unfortunately I don't have these devices or their codes. Have you tried to capture them?

**Comment by Absarking** [comment]
Do you have any idea to capture them

**Comment by probonopd** [comment]
Yes. You need a hardware tool that works with, e.g., IrScrutinizer.
